controller/diary/exercises/deleteExerciseById.js

```diff
@@ -31,6 +29,8 @@ const deleteExerciseById = async (req, res) => {
     { new: true }
   );
 
-  res.status(200).json(data);
+  res
+    .status(200)
```

Comment:
We don’t set the response status to 200, it’s the default

Reply:
fixed